ui/dashboards/src/components/GridLayout/GridTitle.tsx

```diff
@@ -58,30 +56,28 @@ export function GridTitle(props: GridTitleProps) {
         padding: (theme) => theme.spacing(1),
         backgroundColor: (theme) => theme.palette.background.default,
       }}
-      onMouseEnter={() => setIsHovered(true)}
-      onMouseLeave={() => setIsHovered(false)}
     >
       {collapse ? (
         <>
           <IconButton onClick={collapse.onToggleOpen}>
             {collapse.isOpen ? <ExpandedIcon /> : <CollapsedIcon />}
           </IconButton>
           {text}
-          {isEditMode && isHovered && (
+          {isEditMode && (
             <Stack direction="row" sx={{ marginLeft: 'auto' }}>
-              <IconButton aria-label="add panel to group" onClick={openAddPanel}>
-                <AddIcon />
+              <IconButton aria-label={`add panel to group ${title}`} onClick={openAddPanel}>
+                <AddPanelIcon />
```

Comment:
Updated this to match the other icon (at the top of edit mode). I'm checking with Mohit about this change too, but I think he'll be cool with it